resources/views/add_default_user.blade.php

```diff
@@ -2,13 +2,10 @@
 use Illuminate\Support\Facades\App;
 use Illuminate\Support\Facades\DB;
 use Illuminate\Support\Facades\Hash;
-use RonasIT\Support\Traits\MigrationTrait;
 
-class AddDefaultUser extends Migration
+class {{ $migrationName }} extends Migration
```

Comment:
I've checked the latest Laravel migration naming guide and it not using name at all

<img width="446" height="413" alt="Image" src="https://github.com/user-attachments/assets/34a2d520-c31a-4ac8-8da8-fc78e4119092" />

let's use new migration class definition as well